docs/static/security/api-keys.asciidoc

```diff
@@ -2,8 +2,128 @@
 [[ls-api-keys]]
 ==== Grant access using API keys
 
-
 Instead of using usernames and passwords, you can use API keys to grant
 access to {es} resources. You can set API keys to expire at a certain time,
 and you can explicitly invalidate them. Any user with the `manage_api_key`
 or `manage_own_api_key` cluster privilege can create API keys.
+
+{ls} can send both collected data and monitoring information to {es}. If you are
+sending both to the same cluster, you can use the same API key. For different
+clusters, you need to use an API key per cluster.
+
+If you are sending output to different clusters, be sure to create the correct
+kind of API key. 
+
+NOTE: For security reasons, we recommend using a unique API key per {ls} instance.
+You can create as many API keys per user as necessary.
+
+[float]
+[[ls-api-key-publish]]
```

Comment:
@karenzone @colinsurprenant would it make sense to outline a third use case here that creates an API token with read and scan&scroll permissions for the ES input and filter?

Reply:
Would elasticsearch-output be write (publish)? 
And input and filter be `read` and `scan&scroll`?

Reply:
yes, but I am ok if we defer that to another issue to avoid missing train.